integ/validate_cloudwatch/validator.py

```diff
@@ -38,13 +46,54 @@ def log_key_validator(counter, log):
         sys.exit('TEST_FAILURE: found out of order log message')
 
 
+def validate_metric(metric_namespace, dim_key, dim_value, expected_samples=1):
```

Comment:
The other validate function prints "SUCCESS" or "TEST_FAILURE" depending on the outcome; please make this function print that as well.